### DISCUSSION TITLE
AMD PCnet Boot ROM

### DISCUSSION THREAD

**Discussion by davidrg** [86Box/86Box#3475] [Ideas]
**Is your feature request related to a problem? Please describe.**
I've currently got VirtualBox setup to network boot Windows 95 from a NetWare server using an AMD PCnet NIC. Currently it crashes part way through startup and I'm not sure if the problem is VirtualBox or something I've done wrong (its hard finding information on how to do this). It would be good to be able to use 86Box instead to rule out problems with VirtualBox.

**Describe the solution you'd like**
AMD [made a boot ROM image available](https://web.archive.org/web/20010201182300/http://www.amd.com/products/npd/software/pcnet_family/drivers/updates/bootrom.html) for PCnet NICs ([archived here](https://ftp.zx.net.nz/pub/drivers/AMD/pcnet/util/)). I've tested this and it works fine with VirtualBox - would be great if there was some way of using it with 86Box too. 

**Describe alternatives you've considered**
Tried out the Realtek 8029AS card with an RPL boot rom [from here ](https://www.realtek.com/en/component/zoo/category/rtl8029as-software) but it it gives an error when trying to boot (RPL-ROM-ERR: 91 The hardware configuration conflicts; RPL HALTED).

**Additional context**
Screenshots of the PCnet RPL Boot ROM in action [over here](https://www.zx.net.nz/netware/client/dos-netboot/#netboot) booting DOS along with details on the netware side of things.

**Comment by OBattler** [comment]
I have a partial PCnet rewrite planned at some point, I'm going to look into adding the ROM then. Good find!
